docs/src/keymap-upgrade.js

```diff
@@ -0,0 +1,55 @@
+import { Codes, Behaviors } from "./data/keymap-upgrade";
+
+// The start of a behvioar like "&kp" isn't a word boundary,
+// so also match following the start of a line or whitespace.
+const WordBegin = "(?:\\b|^|(?<=\\s))";
+const WordEnd = "\\b";
+const UnimportantWhitespace = "( +)(?!\\S)";
```

Comment:
Should we be using UpperCase?  Any thoughts @Nicell?

Reply:
I personally don't use all upper case on `const`s in my JS code, but it definitely comes down to preference. I generally don't see many JS projects that use upper case for `const`.

Reply:
Fair enough 👍  I'd not seen PascalCase used for this type of variable before which is why it caught my eye.